### DISCUSSION TITLE
[SOLVED] Is installing Node.js required or would unzipping the binary suffice?

### DISCUSSION THREAD

**Discussion by biff80** [martinrotter/rssguard#726] [Q&A] [answered]
I downloaded the portable version of RSSGuard from portable apps. Would unzipping the binary, placing it in RSSGuard's directory and and directing RSS Guard to use it make it truly portable? Or would is it required to install it?

**Comment by martinrotter** [comment, marked as answer]
Hi, I am not sure how node is "installed", if portable installation is supported by nodejs then yes, you can just copy node.exe (npm.exe) and it should work. Make sure to adjust paths in RSS Guard settings and restart app.

**Reply by biff80** [reply]
Thank you for your reply. I thought that I could just unpack Node.js but that doesn't seem to work - I guess I would have to fully install it.

However, I see also that there's two versions of RSS Guard - one with Adblock/WebEngine & one without it. I suppose the version on PortableApps is the one without Adblock/WebEngine. Is it possible to get the portable version of RSS Guard with Adblock/WebEngine on PortableApps?

Alternatively, would it be possible to just copy and paste the requisite files Webengine/adblock files from rssguard-4.2.2-4f2f991b-win64.7z to the PortableApps version? And if so, what directory should I copy over?
